src/Curation/ItemCuration/ItemCuration.model.ts

```diff
@@ -14,4 +15,20 @@ export class ItemCuration extends Model<ItemCurationAttributes> {
       FROM ${raw(this.tableName)} ic
       INNER JOIN ${raw(Item.tableName)} i ON i.id = ic.item_id AND i.collection_id = ${collectionId}`)
   }
+
+  static async getItemCurationCountByThirdPartyId(
+    thirdPartyId: string
+  ): Promise<{ count: number }[]> {
+    return this.query(
+      SQL`SELECT COUNT(DISTINCT ${raw(ItemCuration.tableName)}.id) as Count
+        FROM ${raw(ItemCuration.tableName)}
+        JOIN ${raw(Item.tableName)} ON ${raw(Item.tableName)}.id=${raw(
+        ItemCuration.tableName
+      )}.item_id
+        JOIN ${raw(Collection.tableName)} ON ${raw(
+        Collection.tableName
+      )}.id=${raw(Item.tableName)}.collection_id
+        WHERE ${raw(Collection.tableName)}.third_party_id=${thirdPartyId}`
+    )
```

Comment:
This works perfectly, but to avoid having to be so verbose (`raw`s fault) what do you think about using aliases for the tables and re-using them on the ON and WHERE conditions whenever possible?

Reply:
Oh absolutely, good call @nicosantangelo! Updated to use aliases wherever is possible, looks way better!